core/Symbols.cc

```diff
@@ -1847,7 +1848,6 @@ void ClassOrModule::recordSealedSubclass(MutableContext ctx, ClassOrModuleRef su
     const OrType *orT = nullptr;
     while ((orT = cast_type<OrType>(*iter))) {
         auto right = cast_type_nonnull<ClassType>(orT->right);
-        ENFORCE(left);
```

Comment:
Why is this change here?

Reply:
I thought I committed this to a different branch, but it looks like when I was rebasing it got added to the wrong branch. I’ll fix tomorrow

Reply:
interestingly, this didn't end up committed on two branches, and since it's a no-op I'm just going to land it now so I don't have to wait for a passing rebuild.

Reply:
interestingly, this didn't end up committed on two branches, and since it's a no-op I'm just going to land it now so I don't have to wait for a passing rebuild.